content/sensu-go/5.15/guides/use-federation.md

```diff
@@ -0,0 +1,167 @@
+
+{{< highlight yml >}}
+jwt-private-key-file: /etc/sensu/certs/ec_private.pem
+jwt-public-key-file: /etc/sensu/certs/ec_public.pem
+{{< /highlight >}}
+
+After updating the backend configuration, restart `sensu-backend` for those settings to take effect.
+
+## Step 2 Create etcd replicators
+
+You can use etcd Replicators to synchronize [RBAC policy resources][10] between clusters. This allows you to centrally define permissions that replicate to all federated clusters, ensuring consistent access for Sensu users.
+
+Etcd replicators use the [etcd make-mirror utility][12] for one-way key replication. Our [etcd-replicators reference][2] includes [examples][9] for `Role`, `RoleBinding`, `ClusterRole`, and `ClusterRoleBinding` resources.
+
+## Step 3 Register clusters
+
+Each registered cluster must have a name and a list of cluster member URLs corresponding to the backend REST API.
+
+### Register a single cluster
+
+You do not need to register the cluster that you are currently operating from (for example, self or local-cluster) unless you want to configure the cluster name.
+
+From `local-cluster`, run `sensuctl create` on the yaml or JSON below to register cluster `us-west-2a`:
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+api_version: federation/v1
+type: Cluster
+metadata:
+  name: us-west-2a
+spec:
+  api_urls:
+  - http://10.0.0.1:8080
+  - http://10.0.0.2:8080
+  - http://10.0.0.3:8080
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "Cluster",
+  "metadata": {
+    "name": "us-west-2a"
+  },
+  "spec": {
+    "api-urls": [
+      "http://10.0.0.1:8080",
+      "http://10.0.0.2:8080",
+      "http://10.0.0.3:8080"
+    ]
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+### Register additional clusters
+
+From `local-cluster`, run `sensuctl create` on the yaml or JSON below to register an additional cluster (with the same API URLs as `local-cluster`) and define the name as `us-west-2b`:
+
+{{< language-toggle >}}
+
+{{< highlight yml >}}
+api_version: federation/v1
+type: Cluster
+metadata:
+  name: us-west-2b
+spec:
+  api_urls:
+  - http://10.0.0.4:8080
+  - http://10.0.0.5:8080
+  - http://10.0.0.6:8080
+{{< /highlight >}}
+
+{{< highlight json >}}
+{
+  "api_version": "federation/v1",
+  "type": "Cluster",
+  "metadata": {
+    "name": "us-west-2b"
+  },
+  "spec": {
+    "api-urls": [
+      "http://10.0.0.4:8080",
+      "http://10.0.0.5:8080",
+      "http://10.0.0.6:8080"
+    ]
+  }
+}
+{{< /highlight >}}
+
+{{< /language-toggle >}}
+
+## Step 4 Get a unified view of all your clusters in the web UI
+
+After you create clusters using the federation API, you can log in to the Sensu web UI to view them. In the web UI, you can see the status of every federated cluster, as well as metrics like the number of events and entities for each. 
+
+**NEEDED**: Add screenshots of the context switcher in the web UI showing how the federated views work.
```

Comment:
@jamesdphillips posted this in #web-ui, this could be useful to use as a screenshot. I think he also has some gifs available.
<img width="1085" alt="screen_shot_2019-11-22_at_4 40 58_pm" src="https://user-images.githubusercontent.com/10634078/69981445-2d67ce00-14e7-11ea-931a-573a08475e97.png">

Reply:
I've learned this status display is a "coming soon" feature, so we'll include it when it ships.